komand/server.py

```diff
@@ -98,8 +102,76 @@ def handler(prefix, name, test):
                 r.status_code = status_code
                 return r
 
+        @app.route("/add_worker", methods=["POST"])
```

Comment:
```suggestion
        @app.route("/workers/add", methods=["PUT"])
```
Or perhaps `/workers/increase`